pandas/io/json/normalize.py

```diff
@@ -2,6 +2,7 @@
 # JSON normalization routines
 
 from collections import defaultdict
+from typing import Union, List, Dict, DefaultDict
```

Comment:
The order here I think is causing one of the CI failures. Try `from typing import DefaultDict, Dict, List, Union`